<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AptOfflineQtInstall</class>
 <widget class="QDialog" name="AptOfflineQtInstall">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>463</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>466</width>
    <height>463</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>466</width>
    <height>463</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Install Packages</string>
  </property>
  <widget class="QLineEdit" name="zipFilePath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseFilePathButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startInstallButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Install</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/dialog-ok-apply.png</normaloff>:/icons/icons/dialog-ok-apply.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/dialog-cancel.png</normaloff>:/icons/icons/dialog-cancel.png</iconset>
   </property>
  </widget>
  <widget class="QProgressBar" name="statusProgressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>410</width>
     <height>20</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select the zip file</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="progressStatusDescription">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>53</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="rawLogHolder">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
